superliminal.asl

```diff
@@ -89,6 +106,24 @@ startup
 
     settings.Add("split_ParkingLot", false, "Split on checkpoint \"_ParkingLot\"");
     settings.SetToolTip("split_ParkingLot", "This CP is ignored by default to avoid an inconsistent CP skip.\nEnable this with caution.");
+<<<<<<< HEAD
```

Comment:
Did you test the script after merging? Seems like there are still some merge conflicts

```suggestion
```